src/components/FormPlanning/index.jsx

```diff
@@ -0,0 +1,111 @@
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+import TimePicker from '@mui/lab/TimePicker';
+
+export const FormPlanning = () => {
+    const [date, setDate] = useState(null);
+    const [hour, setHour] = useState(null);
+    const [type, setType] = useState(null);
+    const [platform, setPlatform] = useState(null);
+
+    const handleChange = (event) => {
+        setType(event.target.value);
+    };
+
+    const changePlatform = (event) => {
+        setPlatform(event.target.value);
+    };
+    return (
+        <Box component="form">
+            <TextField id="outlined-basic" label="Interview title" variant="outlined" sx={{ width: '100%', mb: 2 }} />
+            <Box sx={{ mb: 2 }}>
+                <LocalizationProvider dateAdapter={AdapterDateFns} sx={{ mr: 1 }}>
+                    <DatePicker
+                        label="Choose date"
+                        value={date}
+                        onChange={(newValue) => {
+                            setDate(newValue);
+                        }}
+                        renderInput={(params) => <TextField {...params} />}
+                    />
+                </LocalizationProvider>
+                <LocalizationProvider dateAdapter={AdapterDateFns} sx={{ mr: 1 }}>
+                    <TimePicker
+                        label="Choose hour"
+                        value={hour}
+                        onChange={(newValue) => {
+                            setHour(newValue);
+                        }}
+                        renderInput={(params) => <TextField {...params} />}
+                    />
+                </LocalizationProvider>
+                <FormControl sx={{ width: 200 }}>
+                    <InputLabel id="demo-simple-select-label">Interview type</InputLabel>
+                    <Select
+                        labelId="demo-simple-select-label"
+                        id="demo-simple-select"
+                        value={type}
+                        label="Interview type"
+                        onChange={handleChange}
+                    >
+                        <MenuItem value="remote">Remote</MenuItem>
+                        <MenuItem value="presencial">Presencial</MenuItem>
+                    </Select>
+                </FormControl>
+            </Box>
+            <FormControl sx={{ width: '50%', mb: 2 }}>
+                <InputLabel id="demo-simple-select-label">Platform</InputLabel>
+                <Select
+                    labelId="demo-simple-select-label"
+                    id="demo-simple-select"
+                    value={platform}
+                    label="Interview type"
+                    onChange={changePlatform}
+                >
+                    <MenuItem value="zoom">Zoom</MenuItem>
+                    <MenuItem value="google-meet">Google meet</MenuItem>
+                    <MenuItem value="microsoft-teams">Microsoft teams</MenuItem>

```

Comment:
This select is not useful if the recruiter doesn't select the remote interview type